cockatrice/src/tab_supervisor.cpp

```diff
@@ -735,3 +735,18 @@ const ServerInfo_User *TabSupervisor::getOnlineUser(const QString &userName) con
 
     return nullptr;
 };
+
+bool TabSupervisor::switchToGameTabIfAlreadyExists(const int gameId)
+{
+    bool isGameTabExists = false;
+    foreach (const int tabGameId, gameTabs.keys()) {
```

Comment:
Could shortcut this with `if (gameTabs.keys().index(gameId) == -1)` or smth

Reply:
you can simply replace the loop with `gameTabs.contains(gameId)`